internal/model/search_test.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSearch_FilterType(t *testing.T) {
```

Comment:
What's the purpose of this test? I'm puzzled

Reply:
The intention was to have an additional , lets say "safe guard" for developers, not easy to change the order of the enum and give them a hint.
When you create an empty SearchOptions `... := SearchOptions{}`, of course the value zero is the default value for `NameFilterType`.  
If the enum iota order is changed (maybe by some mistake) and the app behavior is not correctly covered by other tests (e.g. by not exact counting of the search result), we could have easily a bug that you dont see on the first view.